.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,32 @@
+name: Feature Request
+description: File a feature request
+title: ":pray: "
```

Comment:
```suggestion
title: ":pray: "
```
What do you think about using 🗳️ here? It gets around the default yellow emoji hands which is a personal pet peeve of mine

Reply:
Absolutely, I'll update this!